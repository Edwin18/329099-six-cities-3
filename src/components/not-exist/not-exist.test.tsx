import * as React from 'react';
import * as renderer from 'react-test-renderer';
import NotExist from './not-exist';
import {Router} from 'react-router-dom';
import history from '../../history';

it(`Render <NotExist />`, () => {
  const tree = renderer
    .create(<Router history={history}>
      <NotExist />
    </Router>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
