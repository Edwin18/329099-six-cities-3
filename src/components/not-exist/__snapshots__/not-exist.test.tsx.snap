// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render <NotExist /> 1`] = `
<div
  className="not-exist"
>
  <p>
    Current offer doesn't exist
  </p>
  <a
    className="not-exist__link"
    href="/"
    onClick={[Function]}
  >
    &gt;Go to main page&lt;
  </a>
</div>
`;
